app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class VotesController < ApplicationController
+
+  def create
+    current_user.votes.create(votable_type: vote_params[0], votable_id: vote_params[1])
```

Comment:
1. what happens if the user has already voted on a thing?
1. reference the params as `vote_params[:votable_type]` and `vote_params[:votable_id]`?  bonus round: just pass `vote_params` to the `create` method?